src/components/sbb-tag-group/sbb-tag-group.tsx

```diff
@@ -107,44 +112,61 @@ export class SbbTagGroup implements ComponentInterface {
     }
   }
 
-  public connectedCallback(): void {
+  public override connectedCallback(): void {
+    super.connectedCallback();
+    const signal = this._abort.signal;
```

Comment:
abortion in disconnectedCallback missing

Reply:
Isn't it already handled in the ReactiveController with the `hostDisconnected()` method?

Reply:
Oh I thought it's a normal AbortController, but it's something new :D Yes, you are absolutely right. Probably we can use that even more often :)